doc/man/zcashd.1

```diff
@@ -84,7 +84,7 @@ Keep at most <n> unconnectable transactions in memory (default: 100)
 .HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-8\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-16\fR to 16, 0 = auto, <0 =
```

Comment:
Where did this change come from? It appears that the default, `DEFAULT_SCRIPTCHECK_THREADS`, is zero, which is auto.

Reply:
The Debian tool is pulling the man page content from the `zcashd` help text. This part of it is the number of threads on the machine running the release script, which is intentionally personalised to the machine because it's user-facing help text. So, this is an artifact of us being lazy when generating the man page (which is not dynamic, unlike the help text).